index.js

```diff
@@ -151,8 +156,8 @@ module.exports = makePaymentChannelPlugin({
     self.authToken = opts.token
     self.settleDelay = opts.settleDelay || DEFAULT_SETTLE_DELAY
 
-    // TODO: figure out best way to create secure keypair
-    self.keyPair = nacl.sign.keyPair.fromSeed(sha256(Buffer.from(self.secret)))
+    const keyPairSeed = hmac(self.secret, 'CHANNEL_KEYS:' + self.peerAddress)
```

Comment:
I would make this constant something like `ilp-plugin-xrp-paychan-channel-keys` so that it really can't conflict with anything else, and I would also put it at the top of the file or into your separate constants file